src/Chapter1.hs

```diff
@@ -490,7 +490,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit n = mod n 10
+lastDigit :: Int -> Int
```

Comment:
This is right!
In Haskell, we usually put the types on the first line and then follow it with the function implementation 🙂